### PR TITLE
Update session timeout from command line

### PR DESCRIPTION
**Pull request by zhx828**
You can set it to -1 to keep session always alive.
This solves issue https://github.com/cBioPortal/cbioportal/issues/4531

# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to master.

# Any screenshots or GIFs?
@cBioPortal/backend

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -204,6 +204,7 @@
     <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
     <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
     <mockito.version>1.10.19</mockito.version>
+    <session.timeout>720</session.timeout>
```
Comment (n1zea144) [inline pom.xml]: Can you prefix this with "tomcat" and move it near the other tomcat property:

[https://github.com/zhx828/cbioportal/blob/999f892fae96698f154752875aad5fd1eaf7d572/pom.xml#L218](https://github.com/zhx828/cbioportal/blob/999f892fae96698f154752875aad5fd1eaf7d572/pom.xml#L218)

Comment (n1zea144) [inline pom.xml]: Can you prefix this with "tomcat" and move it near the other tomcat property?
[https://github.com/zhx828/cbioportal/blob/999f892fae96698f154752875aad5fd1eaf7d572/pom.xml#L218](https://github.com/zhx828/cbioportal/blob/999f892fae96698f154752875aad5fd1eaf7d572/pom.xml#L218)

File: src/main/resources/portal.properties.EXAMPLE [hunk 2]
```diff
@@ -216,3 +216,4 @@ oncoprint.defaultview=patient
 # Disable cache (for non authorized portals this can sometimes avoid running
 # out of memory with relatively little performance loss)
 # mybatis.cache.enabled=false
+
```
Comment (n1zea144) [inline src/main/resources/portal.properties.EXAMPLE]: I missed this - do you want to remove this file from the changeset?

Comment (zhx828) [reply src/main/resources/portal.properties.EXAMPLE]: It probably caused due to my previous implementation. Let me update.

**Pull request merged**